report_thesis/src/sections/baseline.tex

```diff
@@ -17,12 +18,24 @@
 Finally, our original replica used a random train/test split for training, in contrast to the original model's manual curation to ensure representation of extreme compositions in both sets.
 This difference stemmed from the original authors' application of domain expertize in their dataset curation --- a process we could not directly replicate.
 Nevertheless, we found that automatically identifying extreme compositions and ensuring that they were present in both the training and testing sets brought us closer to the original model.
+We chose to pull out the $n$ largest and smallest samples by concentration range, for each oxide, and reserve them for the training set.
+Then we would do a random split on the remaining dataset, such that the final train/test split would be a $80\%/20\%$ split.
 
-With these changes, we have created a more accurate replica of the \gls{moc} model, which we will use as our baseline for the rest of the paper.
-As an additional measure, we have presented these changes to one of the original authors, who confirmed that they were reasonable and in line with the original model's implementation.
+With these changes, we have created a more accurate replica of the \gls{moc} model, which we will use as our baseline for the rest of this paper.
+As an additional measure, we have presented these changes to one of the original authors of \citet{cleggRecalibrationMarsScience2017}, who confirmed that they were reasonable and in line with the original model's implementation.
 
 Table \ref{tab:replica_results_rmses} shows the \gls{rmse}s of the original models and our replicas after these changes.
 Figure~\ref{fig:rmse_histograms} illustrates the distribution of these \gls{rmse}s as a grouped histogram.
+The results show that the \gls{rmse}s of our replicas exhibit similar tendencies to the original models.
+However, in some cases, our replicas have a lower \gls{rmse} than the original models, and in others, they have a higher \gls{rmse}.
+These differences are due to a number of factors.
+
+Firstly, the original models were trained with datasets from 1600mm and 3000mm standoff distances, while we only had access to the 1600mm dataset for our replicas.
+Additionally, we automated the outlier removal for the PLS1-SM phase, unlike the original manual process.
```

Comment:
Which we found had no effect though. Unless you're thinking that it may have had an effect now due to the modified split?

Reply:
It did have some effect. And we have no proof that their manual process had little effect. Therefore, it is a possible significant difference worth mentioning.

Reply:
> It did have some effect.

The largest being a shift from $1.94$ to $1.91$ on $Al_2O_3$, but sure, that's not "no effect" as I said.

> And we have no proof that their manual process had little effect.

Didn't say that

> Therefore, it is a possible significant difference worth mentioning.

Wouldn't say significant, but we can mention it - not a problem at all. I just wanted to hear your thought process here.